modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMFormFieldFactoryHelper.java

```diff
@@ -127,7 +127,9 @@ protected String getDDMFormFieldDataType() {
 
 		Class<?> returnType = _getReturnType();
 
-		if (returnType.isAnnotationPresent(DDMForm.class)) {
+		if (returnType.isAnnotationPresent(DDMForm.class) ||
+			returnType.isAssignableFrom(void.class)) {
```

Comment:
@rodrigopaulino, can you please give me an example on when it's necessary, just for we get into the same page and consolidate the understanding? :slightly_smiling_face: 

Thanks in advance

UPDATE: is it related to the `redirectButton` form field?

Reply:
Sure, @natocesarrego. It's related to the `redirectButton`, but also, without it, saying that a field is `void` would result in it having its data type always set to `string`, which is not something coherent to say about a void field.

Hope this clarifies.

Reply:
Makes sense to me. Thanks. :slightly_smiling_face: